frontend/tests/unit/models/transportation-analysis-test.js

```diff
@@ -1,13 +1,66 @@
-    let model = store.createRecord('transportation-analysis', {});
-    assert.ok(model);
+  test('it correctly calculates square feet and ratios for land use', async function(assert) {
+    let analysisMirage = server.create('transportation-analysis', 'projectLandUse');
+    let project = await this.owner.lookup('service:store').findRecord(
+      'project', analysisMirage.projectId, { include: 'transportation-analysis' }
+    );
+    let analysis = await project.get('transportationAnalysis');
+
+    assert.equal(analysis.residentialUnits, 1000) // project.totalUnits
+    assert.equal(analysis.officeSqFt, 15) // project.commercialLandUse[2].grossSqFt (name: Office)
+    assert.equal(analysis.regionalRetailSqFt, 35) // project.commercialLandUse[3].grossSqFt (name: Regional Retail)
+    assert.equal(analysis.localRetailSqFt, 60) // project.commercialLandUse[4].grossSqFt (name: Local Retail)
+    assert.equal(analysis.restaurantSqFt, 90) // project.commercialLandUse[0].grossSqFt + project.commercialLandUse[1].grossSqFt (name: Fast Food Restaurant, name: Restaurant (not fast food))
+    assert.equal(analysis.communityFacilitySqFt, 100) // project.communityFacilityLandUse[0].grossSqFt (name: General Community Facility)
+    assert.equal(analysis.offStreetParkingSpaces, 170) // project.parkingLandUse[0].spaces + project.parkingLandUse[1].spaces (name: Garages, name: Lots)
+
+    assert.equal(analysis.residentialUnitsRatio, 5); // residentialUnitsRatio = residentialUnits / thresholdFor residentialUnits zone 2 = 1000/200
+    assert.equal(analysis.officeSqFtRatio, 0.00015); // officeSqFtRatio = officeSqFt / thresholdFor officeSqFtRatio zone 2 = 15/100000
+    assert.equal(analysis.regionalRetailSqFtRatio, 0.00175); // regionalRetailSqFtRatio = regionalRetailSqFt / thresholdFor regionalRetailSqFt zone 2 = 35/20000
+    assert.equal(analysis.localRetailSqFtRatio, 0.004); // localRetailSqFtRatio = localRetailSqFt / thresholdFor localRetailSqFt zone 2 =  60/15000
+    assert.equal(analysis.restaurantSqFtRatio, 0.0045); // restaurantSqFt Ratio = restaurantSqFt / thresholdFor restaurantSqFt zone 2 = 90/20000
+    assert.equal(analysis.communityFacilitySqFtRatio, 0.004); // communityFacilitySqFtRatio = communityFacilitySqFt / thresholdFor communityFacilitySqFt zone 2 =  100/25000
+    assert.equal(analysis.offStreetParkingSpacesRatio, 2); // offStreetParkingSpacesRatio = offStreetParkingSpaces / thresholdFor offStreetParkingSpaces zone 2 =  170/85
+  });
+
+  test('it correctly calculates weightedAverage', async function(assert) {
+    let analysisMirage = server.create('transportation-analysis', 'projectLandUse');
+    let project = await this.owner.lookup('service:store').findRecord(
+      'project', analysisMirage.projectId, { include: 'transportation-analysis' }
+    );
+    let analysis = await project.get('transportationAnalysis');
+
+    const sum = 5 + 0.00015 + 0.00175 + 0.004 + 0.0045 + 0.004 + 2; // sum = ~7.0144
```

Comment:
Again, these magic numbers feel kinda brittle ... if the data in the default factory changes your test fails. Maybe just rename the trait to be test-specific, so it's clear that its not just a random trait defining some land use, but the values in it are specifically REQUIRED for these model tests?